nexus/inventory/src/examples.rs

```diff
@@ -321,8 +321,19 @@ pub fn representative() -> Representative {
             slot: 3,
         },
     ];
-    let zpools = vec![];
-    let datasets = vec![];
+    let zpools = vec![InventoryZpool {
```

Comment:
This was another oversight, I really should have added both of these to the representative inventory.

In the future, if anything is added to this representative inventory, but not deleted, the `test_inventory_deletion` test should now help catch it.